cookiecutter/tap-template/{{cookiecutter.tap_id}}/pyproject.toml

```diff
@@ -12,7 +12,7 @@ license = "Apache 2.0"
 [tool.poetry.dependencies]
 python = "<3.11,>=3.7.1"
 requests = "^2.25.1"
-singer-sdk = "^0.13.0"
+singer-sdk = { version="^0.13.0", extras = ["s3"] }
```

Comment:
Wdyt of updating this also on the target SDK template to allow batches files in S3 to be ready by targets?

Reply:
Ah, thanks for catching that!

Reply:
Done